src/models/lineChart.js

```diff
@@ -611,6 +610,8 @@ nv.models.lineChart = function() {
         xTickFormat: {get: function(){return xAxis.tickFormat();}, set: function(_){
             xAxis.tickFormat(_);
             x2Axis.tickFormat(_);
+            tooltip.headerFormatter(_);
+            interactiveLayer.tooltip.headerFormatter(_);
```

Comment:
No, don't do this here.... tooltip.headerFormatter() is already set to a default on line 58, you should just do the same thing with the interactiveLayer's tooltip.   Using the method on line 58 people can update xTickFormat and it'll use that change, but if they manually update headerFormatter then it will keep that so that there aren't issues.